requirements.txt

```diff
@@ -2,7 +2,7 @@
 git+https://github.com/tdryer/hangups.git
 appdirs
 asyncio
-aiohttp
+aiohttp>=1.3,<2 # telepot library also needs this version
```

Comment:
tdryer has [merged the change](https://github.com/tdryer/hangups/pull/320), so this shouldn't be a problem going forward